lib/spack/docs/module_file_support.rst

```diff
@@ -296,65 +271,68 @@ For example, consider the following on one system:
 information associated with the different file formats
 that can be generated by Spack:
 
-  +-----------------------------+--------------------+-------------------------------+----------------------+
-  |                             | **Hook name**      |  **Default root directory**   | **Compatible tools** |
-  +=============================+====================+===============================+======================+
-  |  **Dotkit**                 | ``dotkit``         |   share/spack/dotkit          |  DotKit              |
-  +-----------------------------+--------------------+-------------------------------+----------------------+
-  |  **TCL - Non-Hierarchical** | ``tcl``            |  share/spack/modules          | Env. Modules/LMod    |
-  +-----------------------------+--------------------+-------------------------------+----------------------+
-  |  **Lua - Hierarchical**     | ``lmod``           |   share/spack/lmod            | LMod                 |
-  +-----------------------------+--------------------+-------------------------------+----------------------+
+  +-----------------------------+--------------------+-------------------------------+----------------------------------+----------------------+
+  |                             | **Hook name**      |  **Default root directory**   | **Default template file**        | **Compatible tools** |
+  +=============================+====================+===============================+==================================+======================+
+  |  **Dotkit**                 | ``dotkit``         |  share/spack/dotkit           | templates/modules/modulefile.dk  | DotKit               |
+  +-----------------------------+--------------------+-------------------------------+----------------------------------+----------------------+
+  |  **TCL - Non-Hierarchical** | ``tcl``            |  share/spack/modules          | templates/modules/modulefile.tcl | Env. Modules/LMod    |
+  +-----------------------------+--------------------+-------------------------------+----------------------------------+----------------------+
+  |  **Lua - Hierarchical**     | ``lmod``           |  share/spack/lmod             | templates/modules/modulefile.lua | LMod                 |
+  +-----------------------------+--------------------+-------------------------------+----------------------------------+----------------------+
 
 
-Though Spack ships with sensible defaults for the generation of module files,
-one can customize many aspects of it to accommodate package or site specific needs.
-These customizations are enabled by either:
+Spack ships with sensible defaults for the generation of module files, but
+you can customize many aspects of it to accommodate package or site specific needs.
+In general you can override or extend the default behavior by:
 
  1. overriding certain callback APIs in the Python packages
  2. writing specific rules in the ``modules.yaml`` configuration file
```

Comment:
Ehehe, sometimes you have 15.000 lines in front of you an you don't see them....